tests/network-tests/src/fixtures/membership/BuyMembershipWithInsufficienFundsFixture.ts

```diff
@@ -37,7 +37,7 @@ export class BuyMembershipWithInsufficienFundsFixture extends BaseFixture {
 
     assert.isBelow(
       balance.toNumber(),
-      membershipFee.add(membershipTransactionFee).toNumber(),
+      membershipFee.add(membershipTransactionFee).add(this.api.consts.balances.existentialDeposit).toNumber(),
```

Comment:
I'm adding it because the account is initialized with ed. Are you suggesting removing this addition?

Or is your point that the implementation of buy_membership() is flawed?

Reply:
All I'm saying is that the if the account has a balance that is >= `membershipFee + membershipTransactionFee`, but `< membershipFee + membershipTransactionFee + existentialDeposit` the tx will still pass, so I don't think the assertion is correctly checking the balance before testing for expected failure.

You could say the implementation is a little flawed too, because if someone has a balance `== membershipFee + membershipTransactionFee` then he will only pay `membershipFee + membershipTransactionFee - existentialDeposit`, so slightly less than someone who has a higher balance, because slashing will always leave at least `existentialDeposit` funds in the account and since in the runtime we'll be trying to perform a slash that would result in balance == 0, the `existentialDeposit` part of the slash will just be ignored (returned as `not_slashed`, which is a second value in the tuple retruned by `balances::slash`)

Reply:
That property of `balances::slash` may actually have been missed in some other places in the runtime (for example, when buying nfts), I'll create a separate issue about it

Reply:
Issue mentioned above https://github.com/Joystream/joystream/issues/4105